### ISSUE TITLE
Can't open "Table - Full Page" page

### ISSUE THREAD

**Issue by ShangHungTsai** [frencojobs/vscode-notion#13]
I very love this Plugin, but there have some problem on my Notion page when i open in vscode.

It's look like If the page is "Table - Full Page" it cant be load.

Whether i use it in the wrong way?

**Comment by frencojobs** [comment]
Yep, I don't think the current API can handle the database pages including tables and other structures. I'm planning to migrate the whole thing to the official beta API soon.
